frontend/www/js/omegaup/typeahead.ts

```diff
@@ -100,6 +100,43 @@ function typeahead<T extends { label: string; value: string }>(
     .trigger('change');
 }
 
+export function nominationTypeahead(
+  elem: JQuery<HTMLElement>,
+  cb?: CallbackType<types.NominationListItem>,
+) {
+  if (!cb) {
+    cb = (event: Event, val: types.NominationListItem) =>
+      $(<EventTarget>event.target).val(val.problem.alias);
+  }
+  elem
+    .typeahead<types.NominationListItem>(
+      {
+        minLength: 3,
+        highlight: false,
+      },
+      {
+        source: typeaheadWrapper(
+          (options: { query: string }) =>
+            new Promise<types.NominationListItem[]>((resolve, reject) =>
+              api.QualityNomination.list({ rowcount: 1000, status: 'all' })
```

Comment:
` rowcount`  y ` status`  no son tan necesarios de enviar, ya que tienen valores default y parece que son los mismos que envías. Los que si son necesarios para poder filtrar son `query` y ` column`, así que puedes cambiarlo por: 
```suggestion
              api.QualityNomination.list({
                query: options.query,
                column: column,
              })
```

Reply:
@pabo99 Lo que intento es traer un array con las nominaciones para asi poder traer una mayor cantidad para el autocompletado